src/index.template.html

```diff
@@ -1,17 +1,18 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= productName %></title>
-    <meta name="description" content="<%= productDescription %>">
+    <title><%= title %></title>
```

Comment:
I think `productName` is still OK for this part - once the app loads we [set the title dynamically](https://github.com/yunity/karrot-frontend/blob/4b69cf31d55fac6fba3da27c1f8f9826b95cbd76/src/base/datastore/routerPlugin.js#L107)